Add missing Install requirement to install `cviebrock/eloquent-sluggable` (#409)
Laravel-Backpack/docs: 5.x/crud-operation-update.md

Change the languages available to translate to/from, in your crud config file (```config/backpack/crud.php```). By default there are quite a few enabled (English, French, German, Italian, Romanian).

> Additionally, if you have slugs (but only if you need translatable slugs), you'll need to use backpack's classes instead of the ones provided by cviebrock/eloquent-sluggable and you should install `cviebrock/eloquent-sluggable` as well with `composer require cviebrock/eloquent-sluggable`:

Reviewer: ```suggestion
Additionally, if you have slugs (but only if you need translatable slugs), you'll need to use backpack's classes instead of the ones provided by `cviebrock/eloquent-sluggable`. 
Make sure you have `cviebrock/eloquent-sluggable` installed as well, if not, please do it with `composer require cviebrock/eloquent-sluggable`:
```
